Process.rb

```diff
@@ -0,0 +1,29 @@
+class ProcessManager # I wanted to name this class 'Process', but it seems it is a reserved word
+    # Returns a process instance based on any line read from the input.txt
+    def initialize(line, id)
+        tokens = line.split(',')
+        @process_id = id
+        @entry_time = tokens[0]
+        @priority = tokens[1]
+        @block_size = tokens[3]
+        @printer_requested = tokens[4]
+        @scanner_requested = tokens[5]
+        @modem_requested = tokens[6]
+        @disk_requested = tokens[7]
+
+        @memory_offset = "?????" # I actually don't know what that's supposed to be
```

Comment:
Isn't the memory amount reserved for the process?

Reply:
@mateusluizfb i don't think so. The block length already indicates the memory amount required. Maybe the offset is used to indicate where it is "physically" on the memory but i'm not sure.

Reply:
Ok, I think we can remove then, what you think? If we need it in the future, I think we can add again